packages/stream-model-handler/src/schema-utils.ts

```diff
@@ -13,12 +40,13 @@ export class SchemaValidation {
   })
 
   public async validateSchema(
```

Comment:
Where is this method called? If it's used in TileDocuments then this might break old schemas?

Reply:
this file is specific to the `stream-model-handler` package.  TileDocument has its own schema validation code that is separate

Reply:
ok, ty!